www/index.php

```diff
@@ -495,7 +495,8 @@
 					<div id="unitsQuestionMark" class="questionMark"></div>
 					<label id="selectUnitsControl">Units
 						<select id="selectUnits" data-dojo-type="dijit/form/Select">
-							<option value='defaultSelect'>No Units</option>
+							<option value='defaultSelect'>--Select--</option>
```

Comment:
Can you add a "hidden" property to the -- Select -- option so it doesn't appear in the drop down?